docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -56,3 +56,28 @@ values will be skipped entirely.
 If there are missing values in feature fields (fields that are subjects of the 
 {dfanalytics}), then the document that contains the fields with the missing 
 values will be skipped during the analysis.
+
+[float]
+[[dfa-field-type-docs-limitations]]
+=== {dfanalytics-cap} field type and document limitation
+
+[float]
+==== {oldetection-cap}
+
+{oldetection-cap} requires numeric or boolean data to analyze. The algorithms 
+don't support missing values (see also <<dfa-missing-fields-limitations>>), 
+therefore fields other than numeric or boolean will be ignored. Documents where 
+included fields contain missing values, null values, or an array also will be 
+ignored. Therefore a `dest` index may contain documents that don't have an 
```

Comment:
We don't refer to the "dest" keyword anywhere else in this page, so I think it's better to use the full name...
```suggestion
ignored. Therefore a destination index may contain documents that don't have an 
```